extensions/amp-story-captions/0.1/amp-story-captions-presets.css

```diff
@@ -0,0 +1,48 @@
+@import '../../amp-story/1.0/amp-story-shadow-reset.css';
+
+/* default */
+
+.amp-story-captions-default .amp-story-captions-cue-wrapper {
+  font-size: calc(2.5 * var(--story-page-vh, 8px)) !important;
+  font-family: 'Roboto', sans-serif !important;
+  margin: 1em 0 !important;
+  display: flex !important;
+  justify-content: center !important;
+  flex-direction: column !important;
+  align-items: center !important;
+  width: 100% !important;
+}
+
+.amp-story-captions-default .amp-story-captions-cue {
+  text-align: center !important;
+  color: #fff !important;
+  border-radius: .94em !important;
+  padding: 0.35em 0.94em !important;
+  background-color: hsla(0, 0%, 10%, 0.6) !important;
+  max-width: calc(var(--story-page-vw) * 92) !important;
+}
+
+.amp-story-captions-default .amp-story-captions-future {
+  color: #fff !important;
+}
+
+/* fade in */
+
+.amp-story-captions-appear .amp-story-captions-cue-wrapper  {
+  font-size: var(--story-captions-font-size, calc(2.5 * var(--story-page-vh, 8px))) !important;
+  font-family: var(--story-captions-font-family, 'Roboto', sans-serif) !important; 
```

Comment:
Does this triple comma work for optional values? Also I don't think this would work well if someone specifies the font as a font that doesn't exist (meaning it wouldn't fallback to Roboto).

The right property value would be sth like `var(--story-captions-font-family, 'Roboto'), 'Roboto', sans-serif` so that the font defaults to Roboto if the font is not defined (that's what the first Roboto is for), or if it's not loaded (what the second Roboto is for)

Reply:
Thank you for catching this! Just tested it as you specified and it works :)